### ISSUE TITLE
Blank screen

### ISSUE THREAD

**Issue by dapper27** [Kemmey/Kemmey-TeslaWatch-Public#223]
![image](https://user-images.githubusercontent.com/44276659/100268344-c4c1fe80-2f4c-11eb-8e0e-23443523f362.png)

This is what the app is displaying at the moment!

**Comment by remy911** [comment]
Yes, mine too

**Comment by fddc99** [comment]
same here both 380 and 381 builds lead to blank screen. Complications do display, but app opens to blank. tried force quite watch app, iOS app, delete & re-install. restart both devices no luck.
Was able to get back to app by reverting to 365 on my watch series 6

**Comment by Kemmey** [comment]
Yeah, sorry about that - simple bug on my part - new build going out in a few minutes!

**Comment by fddc99** [comment]
👌No problem... gets better every build. Any chance the app will be offered as Mac app for M1 silicon?

**Comment by dapper27** [comment]
I've accidentally left Testflight. How do I get back to the beta testing?

**Comment by dapper27** [comment]
It's ok, ignore that, I found the email with the invite.

**Comment by dapper27** [comment]
Just got the latest build. Working fine. Thanks for the quick rollout!

**Comment by Kemmey** [comment]
@fddc99 The current release has been opted out of macOS until I receive my own m1 machine. I've recently added iPad support - so should be sort-of ready for macOS. But to do it properly, which would involve a menubar icon and mini-popover, will take some time. I don't think it'll be very useful, except maybe for the widget. But I'll see how much work it is when I get my m1. Should be here mid December.
